pkg/validation/validate.go

```diff
@@ -135,11 +149,23 @@ func ValidateLabels(limits LabelValidationLimits, tenantID string, ls []*typesv1
 		if len(l.Value) > limits.MaxLabelValueLength(tenantID) {
 			return NewErrorf(LabelValueTooLong, LabelValueTooLongErrorMsg, phlaremodel.LabelPairsString(ls), l.Value)
 		}
-		var origName string
+		var origName, sanitized string
 		var ok bool
-		if origName, l.Name, ok = SanitizeLabelName(l.Name); !ok {
+		if origName, sanitized, ok = SanitizeLabelName(l.Name); !ok {
```

Comment:
I find this a lot more readable
```suggestion
origName, sanitized, ok := SanitizeLabelName(l.Name)
if !ok {
```